### PR TITLE
[Test_Chris] Helloworld PR

### PR DESCRIPTION
**Pull request by ChristopherCao**
### What?

- brief: a small try for creating a team folder and add some codes
- author: Chris

### What changes are included in this PR?
added a helloworld cpp file

### CODE REVIEW

**Comment by Light-City** [comment]
The directory structure is incorrect, you can refer to what others have done, or this document

https://github.com/datawhalechina/whale-starry/blob/main/stl/rules/pr.md

**Comment by ChristopherCao** [comment]
Sorry for the incorrection.
Here is the correct picture link.
https://github.com/ChristopherCao/whale-starry/blob/main/result.png

**Comment by ChristopherCao** [comment]
Sorry for the incorrection.
Here is the correct picture link.
https://github.com/ChristopherCao/whale-starry/blob/main/final_result.png

File: stl/Chris/src/helloworld
Comment (martinx) [inline stl/Chris/src/helloworld:1]: remove the binary file

File: stl/Chris/src/helloworld.cpp
Comment (martinx) [inline stl/Chris/src/helloworld.cpp:1]: Please write code following the  Google Code Style

1: `{` do not need to start a new line
2: Indent two spaces

**Review by martinx — COMMENTED** [review]
Please use lowercase

Chris => chris

**Comment by martinx** [comment]
> The directory structure is incorrect, you can refer to what others have done, or this document
> 
> https://github.com/datawhalechina/whale-starry/blob/main/stl/rules/pr.md

Please refer to this to refactor the directory structure.

**Comment by ChristopherCao** [comment]
Oh sorry.
I have corrected all the issues you've mentioned above.
Here is my final version.
Wish you a good day.
https://github.com/ChristopherCao/whale-starry/blob/main/pr.png

File: stl/chris_team/chris/src/helloworld.cpp [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#include <iostream>
+
+int main()  {
```
Comment (martinx) [inline stl/chris_team/chris/src/helloworld.cpp:3]:
```cpp
int main()  { => int main() {
```

1 space

File: stl/chris_team/chris/src/helloworld.cpp [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+    std::cout << "Helloworld!" << std::endl;
```
Comment (martinx) [inline stl/chris_team/chris/src/helloworld.cpp:4]: Indent two spaces

**Comment by ChristopherCao** [comment]
Sorry, I checked google code style and make sure this commit will be perfect.
Here is the pic. link.
https://github.com/ChristopherCao/whale-starry/blob/main/final_pr.png
Thanks for ur hard work!

**Comment by martinx** [comment]
LGTM

**Pull request closed without merging**